CMSgov/cmcs-eregulations
# Ereggcsc 1760 - metadata full text resources results - (original had problems)

Resolves #[EREGCSC-1760](https://jiraent.cms.gov/browse/EREGCSC-1760)

Description-
Setup resources to query search.gov and from the list of results get the rest of the data from our database.
Add snippet to the result set. But for now the ui will just show description. Our next step we will display both snippet and description.

This pull request changes...

update resources.py
add a route in urls.py
Create serializer to return search.gov's snippets that we apply to the response
Update api.js so that if it finds searchgov in the url to call the route path that looks it up with searchgov, otherwise to call the original route path.
Steps to manually verify this change...

steps to view and verify change
Visit https://dv9wcjg6r3.execute-api.us-east-1.amazonaws.com/dev740/v3/resources/search?q=medical
Notice the data contains "Snippet" and "Description"
visit https://dv9wcjg6r3.execute-api.us-east-1.amazonaws.com/dev740/search/?q=medical&searchgov
notice the url has searchgov so the page will make a request to search.gov. Notice description is there but snippet is not there yet. (You can saw that in the api call in step 1)
click next page and check see if pagination working correctly
visit https://dv9wcjg6r3.execute-api.us-east-1.amazonaws.com/dev740/search/?q=ex%20parte
notice only 1 search result which is coming from our db
Visit https://dv9wcjg6r3.execute-api.us-east-1.amazonaws.com/dev740/search/?q=ex%20parte&searchgov
notice lots more results as search.gov can retrieve results from pdf files as well.
